ci/terraform/waf.tf

```diff
@@ -169,6 +199,25 @@ resource "aws_wafv2_web_acl" "frontend_alb_waf_regional_web_acl" {
     }
   }
 
+  rule {
+    action {
+      block {}
+    }
+    priority = 60
```

Comment:
This will lower the priority for Other Env also ? Since  AWS WAF evaluates  from the lowest numeric priority setting 

May be logic to set highest priority if environment is staging ?  leaving the priority untouched for other Env

Reply:
Yes, fair point. Changed, thanks